lib/advisor/core/questionnaire/form.ex

```diff
@@ -1,8 +1,8 @@
 defmodule Advisor.Core.Questionnaire.Form do
   alias Advisor.Core.{People, Questions}
 
-  def data() do
-    everybody = People.everybody()
+  def data_of(connected_user) do
```

Comment:
As this method is only used in this specific case, it seemed legit to me to name the variable related to its case. The `connected_` here avoid having to research, who is the data for, who isn't part of _everybody_